assets/src/scripts/monitoring-location/iv-data-utils.js

```diff
@@ -155,7 +155,8 @@ const getConvertedTimeSeries = function(timeSeries, tsRequestKey, parameterCode)
             return {
                 ...point,
                 qualifiers: [...point.qualifiers],
-                value: point.value ? convertCelsiusToFahrenheit(point.value).toFixed(2) : null
+                value: Number.isNaN(point.value) || point.value === '' || isNaN(point.value) ?
```

Comment:
Are both Number.isNaN and isNaN needed?

Reply:
Well, it looks like Number. IsNaN isn't needed. JavaScript can certainly be squarely with truthful evaluations. Number. IsNaN wouldn't return null for a string in the above code, it would return NaN. Which means it evaluated as true, then ran the convert function which returned NaN, Weird. But I was more surprised that an empty string, would return 32 degrees. I guess in JavaScript land a blank string is a numerical zero. Just some of many reasons JavaScript is not idea for calculations.